sphinx/setup_command.py

```diff
@@ -144,15 +145,19 @@ def finalize_options(self):
             self.config_dir = self.source_dir
         self.config_dir = abspath(self.config_dir)
 
+        self.ensure_string_list('builder')
         if self.build_dir is None:
             build = self.get_finalized_command('build')  # type: ignore
             self.build_dir = os.path.join(abspath(build.build_base), 'sphinx')
             self.mkpath(self.build_dir)  # type: ignore
         self.build_dir = abspath(self.build_dir)
         self.doctree_dir = os.path.join(self.build_dir, 'doctrees')
         self.mkpath(self.doctree_dir)  # type: ignore
-        self.builder_target_dir = os.path.join(self.build_dir, self.builder)
-        self.mkpath(self.builder_target_dir)  # type: ignore
+        self.builder_target_dirs = [
+            (builder, os.path.join(self.build_dir, builder))
+            for builder in self.builder]  # type: List[(str, str)]
```

Comment:
This type annotation is invalid. You should use `List[Tuple[str, str]]` instead.
https://travis-ci.org/sphinx-doc/sphinx/jobs/205824862#L594

BTW, I think the second element of the tuple is a unicode string.
Certainly `builder` is raw string, but `self.build_dir` is a unicode (because we can build docs on path including multibytes). As a result, the joined string becomes a unicode string.